File: preliminary_analysis/baseline_cpp_results/cpp_source/question-44-pass-0.cpp
// Digitally balanced numbers. 
 #include <bits/stdc++.h> 
 using namespace std ;
 int checkSame ( int n , int b ) {
 
 int temp = n ;
 int count = 0 ;
 while ( temp > 0 ) {
 int rem = temp % b ;
 if ( rem == 0 )
 count++ ;
 temp /= b ;
 }
 return count ;
 }
 int main ( ) {
 int n ;
 cin >> n ;
 int b ;
 cin >> b ;
 int count = 0 ;
 for ( int i = 1 ; i <= n ; i++ ) {
 if ( checkSame ( i, b ) == checkSame ( n - i, b ) )
 count++ ;
 }
 

File: results/preliminary_analysis/baseline_py_results/py_source/question-44-pass-0.py
# Digitally balanced numbers. 
def checkSame(n, b): 
    if n == 0: 
        return True
    if n < 0: 
        return False
    if n % b == 0: 
        return False
    return checkSame(n // b, b) 
  
# Driver Code 
n = 1234567890 
b = 10
print(checkSame(n, b)) 
  
# This code is contributed by Nikhil Kumar Singh 